scripts/cmake/ProcessesSetup.cmake

```diff
@@ -2,7 +2,7 @@
 # processes to be build.
 set(ProcessesList
     ComponentTransport
-    GroundwaterFlow
+    SteadyStateDiffusion
```

Comment:
Can you put it in more alphabetic order?

Reply:
I wonder why 'sed' didn't reorder it automatically :smiley:  - I will do it for 'sed'.